app/config/set_parameters.php

```diff
@@ -4,9 +4,15 @@
 
 $parametersFilepath = __DIR__  . '/parameters.php';
 if (!file_exists($parametersFilepath)) {
+    // let's check first if there's some old config files which could be migrated
     Upgrade::migrateSettingsFile();
 }
 
+if (!file_exists($parametersFilepath)) {
```

Comment:
Hello,

Instead of checking two times the same file, you could return if the result of `Upgrade::migrateSettingsFile();` is false.

```php
 if (!file_exists($parametersFilepath)) {
    // let's check first if there's some old config files which could be migrated
    if (Upgrade::migrateSettingsFile() === false) {
         return;
    }
 }
```